mixed/spark/common/src/main/java/com/netease/arctic/spark/mixed/ArcticDataSource.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.spark.mixed;
+
+/** Arctic DataSource register */
+public class ArcticDataSource extends MixedSourceBase {
+
+  @Override
+  public String shortName() {
+    return "arctic";
```

Comment:
Is `arctic` still a proper name for the source? We have changed the project name to amoro and the project name is no longer related to a specific source.

Reply:
fixed